Automattic/wp-calypso
# Newsletter Categories: Add mutation for unmarking newsletter category

Closes https://github.com/Automattic/wp-calypso/issues/80179

## Proposed Changes

This PR adds a mutation to use the endpoint to unmark a category as "Newsletter Category". 

## Testing Instructions

You can test this by running the tests: in your terminal, execute `yarn test-client client/data/newsletter-categories/test/use-unset-newsletter-category-mutation.test.tsx`. All the tests should be green.

## Pre-merge Checklist
